### PR TITLE
Pocket Core improvements

### PR DESCRIPTION
**Pull request by pabelnl**
removed relay property from Pocket, added relay data checks for json to stringify

### CODE REVIEW

File: packages/core/src/core.js [hunk 1]
```diff
@@ -65,6 +64,12 @@ class Pocket {
 
   // Create a Relay instance
   createRelay(blockchain, netID, data) {
+    // Check if data is a json tring
+    if (typeof data == 'string') {
+      return new Relay(blockchain, netID, data, this.configuration);
+      
+    }
+    data = JSON.stringify(data);
```
Comment (luyzdeleon) [inline packages/core/src/core.js]: This should be an else: 

`return  new Relay(blockchain, netID, JSON.stringify(data), this.configuration);`

File: packages/core/src/core.js [hunk 2]
```diff
@@ -178,7 +183,7 @@ class Pocket {
 
       if (node == null) {
         if (callback) {
-          callback(null, new Error("Node is empty;"));
+          callback(new Error("Node is empty;"));
```
Comment (luyzdeleon) [inline packages/core/src/core.js]: Change `;` to `.`

File: packages/core/src/models/wallet.js [hunk 3]
```diff
@@ -1,11 +1,14 @@
-
 // Wallet - Model
-function Wallet(address, privateKey, network, subnetwork, data) {
-  this.address = address;
-	this.privateKey = privateKey;
-	this.network = network;
-	this.subnetwork = subnetwork;
-	this.data = data;
+class Wallet {
+	constructor(address, privateKey, network, subnetwork, data) {
```
Comment (luyzdeleon) [inline packages/core/src/models/wallet.js:3]: Change subnetwork to netID or networkID

**Review by luyzdeleon — CHANGES_REQUESTED** [review]

File: packages/core/src/core.js (hunk 1 above)
Comment (pabelnl) [reply packages/core/src/core.js]: no need to encapsulate in an else, the previous statement already does a return even if is an error or Relay.

Comment (luyzdeleon) [reply packages/core/src/core.js]: Yeah but no need to reassign the data variable to just return it in the next line, agreed with the else.

**Review by luyzdeleon — APPROVED** [review]

**Pull request merged**